docs/release-notes.md

```diff
@@ -97,6 +97,24 @@ simply executing `qx test`, using pluggable test runners.
 
 [Documentation](https://qooxdoo.org/documentation/#/development/testing/)
 
+## Component behaviour changes
+
+In hindsight of better keyboard accessibility support some components received changes regarding their behaviour:
+
+| Component | Changes |
+| --- | --- |
+| qx.ui.form.AbstractSelectBox | `Tab` Key is now equally treated as `Escape` and closes the menu |
+| qx.ui.form.ComboBox | - `Down` Key now opens the menu without pressing `Alt` <br /> - Textfield text is not fully selected anymore when the menu closes <br /> - Pressing input specific keys while the menu is open will close the menu|
```

Comment:
did'nt you mean "is fully selecteded"?

Reply:
No, all text is currently getting selected when the menu closes. This got changed in the PR. tabFocus is no longer called in handler _onPopupChangeVisibility